src/problem-analysis/presentation.tex

```diff
@@ -0,0 +1,27 @@
+% textidote: ignore begin
+\section{Presentation of the problem}\label{sec:presentation-of-the-problem}
+% textidote: ignore end
+
+Several shortcomings from NOVA have been documented in the transcript below.
+The most noticeable is the issues regarding the~\acrshort{epos} insights and how they are not elaborate enough in its
+current state.
+
+% textidote: ignore begin
+\begin{displayquote}
+% textidote: ignore end
+    \large\textbf{“}As NOVA is a relatively new business, a lot of minor as well as major issues have to be solved.
```

Comment:
@MadsHeilmann https://tex.stackexchange.com/a/135847/305936
https://mirrors.dotsrc.org/ctan/macros/latex/contrib/csquotes/csquotes.pdf
Also, it's never a good idea to force unicode characters

Reply:
Agree. Has been fixed